data/layer-groups/zoning-for-accessibility.json

```diff
@@ -0,0 +1,62 @@
+{
+  "id": "zoning-for-accessibility",
+  "legend": {
+    "label": "Zoning For Accessibility",
+    "tooltip": "Transit-adjacent areas where applicable developments and enlargements require additional consultation with the MTA or relevant transit agencies.",
+    "infolink": "https://www.nyc.gov/content/planning/pages/our-work/plans/citywide/elevate-transit-zoning-accessibility"
+  },
+  "layers": [
+    {
+      "before": "place_state",
+      "style": {
+        "id": "zfa-line",
+        "type": "line",
+        "source": "supporting-zoning",
+        "source-layer": "zoning-for-accessibility",
+        "paint": {
+          "line-width": {
+            "stops": [
+              [
+                11,
+                1
+              ],
+              [
+                12,
+                3
+              ]
+            ]
+          },
+          "line-color": "#0039A6"
+        }
+      }
+    },
+    {
+      "highlightable": true,
+      "clickable": true,
+      "tooltipable": true,
+      "tooltipTemplate": "Transit Buffer {{{station_na}}}",
```

Comment:
I think you're missing a hyphen here
<img width="771" alt="image" src="https://github.com/user-attachments/assets/67802732-71a5-4e21-a10b-57cbe35b1fd1" />